src/ripple/app/ledger/LedgerTiming.h

```diff
@@ -83,6 +83,9 @@ const int AV_STUCK_CONSENSUS_PCT = 95;
 
 const int AV_CT_CONSENSUS_PCT = 75;
```

Comment:
So what's `AV_CT_CONSENSUS_PCT` anyway?  Anything involving consensus is of critical importance in correctness - if you know what this does, perhaps you could add a comment here?